app/models/google_calendar/google_event.rb

```diff
@@ -21,6 +21,12 @@ def find_by_room(time_interval, conference_room_ids)
       event_finder.by_room(time_interval, conference_room_ids)
     end
 
+    def confirm(conference_room_id, event_id)
+      conference_room = ConferenceRoom.find(conference_room_id)
```

Comment:
Im wondering...

```ruby
Event.find_or_create_by(conference_room_id: conference_room_id, event_id: event_id) do |event| 
  event.confirm
end
```

Will do the job? And do you really need to create an event here? Shouldn't it be created before?